packages/web-components/src/components/va-segmented-progress-bar/va-segmented-progress-bar.tsx

```diff
@@ -109,10 +122,10 @@ export class VaSegmentedProgressBar {
 
       return (
         <Host>
-          <div class={indicatorClass} aria-label={label}>
```

Comment:
**Should 1:** Remove aria-label from `usa-step-indicator` as it's a non-semantic element ✅